README.fr.md

```diff
@@ -0,0 +1,122 @@
+[![Build Status](https://travis-ci.org/osmlab/name-suggestion-index.svg?branch=master)](https://travis-ci.org/osmlab/name-suggestion-index)
+[![npm version](https://badge.fury.io/js/name-suggestion-index.svg)](https://badge.fury.io/js/name-suggestion-index)
+
+## nom-suggestion-index 
```

Comment:
This should probably just say `name-suggestion-index` since it's the actual name of the project.

Reply:
yes, projects name must not be translated.